dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -41,7 +80,7 @@ CUSTOM_CERTS_DIR=/public-certs
 # Check for additional CA certificates propagated to Keycloak
 if [[ -d $CUSTOM_CERTS_DIR && -n $(find ${CUSTOM_CERTS_DIR} -type f) ]]; then
     for certfile in ${CUSTOM_CERTS_DIR}/* ; do
-        keytool -importcert -alias CERT_$(basename $certfile) -keystore $KEYSTORE_PATH -file $certfile -storepass $TRUST_STORE_PASSWORD  -noprompt;
+        jks_import_ca_bundle $certfile $KEYSTORE_PATH $TRUST_STORE_PASSWORD
```

Comment:
shellcheck will complain here, asking that you wrap the vars in quotes to avoid space-splitting in paths... 

https://www.shellcheck.net/

![image](https://user-images.githubusercontent.com/227597/101659295-e672b980-3a13-11eb-821d-df6f88a66f9f.png)

Reply:
I see, thank you for pointing out.